src/utils/getSortedPeople.ts

```diff
@@ -0,0 +1,42 @@
+import { Person } from '../types';
+
+export const getSortedPeople = (
+  people: Person[],
+  sortBy: string | null,
+  order: string | null,
+) => {
+  const sortedPeople = people;
+
+  if (sortBy) {
+    if (sortBy) {
```

Comment:
Twice the same condition? Looks like a typo :)

You can avoid any if statement here by just providing a `default` case for switch statement (btw, you should add a default case anyways).